aem-classification-validator/src/main/java/biz/netcentric/filevault/validator/aem/classification/AemClassificationValidatorFactory.java

```diff
@@ -46,7 +47,9 @@
      * resource based maps. Also all other known protocols are supported */
     private static final String OPTION_MAPS = "maps";
     /** optional list of comma-separated resource path patterns (should be absolute) */
-    private static final String OPTION_WHITELISTED_RESOURCE_PATH_PATTERNS = "whitelistedResourcePathsPatterns";
+    private static final String OPTION_WHITELISTED_RESOURCE_PATH_PATTERNS = "whitelistedResourcePathPatterns";
+    private static final String OPTION_WHITELISTED_RESOURCE_PATH_PATTERNS_OLD = "whitelistedResourcePathsPatterns";
```

Comment:
Considering that it's early on since the first release I would maybe keep it simple and just not support the old property name